packages/devtools/src/index.ts

```diff
@@ -0,0 +1,44 @@
+import devtools from '@vue/devtools'
+import express from 'express'
+// @ts-ignore
+import launchMiddleware from 'launch-editor-middleware'
+
+export interface DevtoolsOptions {
+  title?: string
+  host?: string
+  openInEditor?: boolean
+}
+
+const SERVER_PORT = 8098
+
+export function createDevtools(options: DevtoolsOptions = {}) {
+  const {
+    host = 'http://localhost',
+    openInEditor = true,
+    title = 'vue-termui devtools',
+  } = options
+
+  // workaround for @vue/devtools
+  global.document = {
+    title,
+    createElement: () => ({}),
+    querySelector: () => ({}),
+    querySelectorAll: () => [],
+  }
+
+  if (openInEditor) {
+    global.VUE_DEVTOOLS_CONFIG = {
+      openInEditorHost: `${host}:${SERVER_PORT}/`,
+    }
+    const app = express()
+    app.use('/__open-in-editor', launchMiddleware())
+    app.listen(SERVER_PORT)
```

Comment:
@posva 

I reused the `@vue/devtools` server instead of create new server.

I think reused the `@vue/devtools` server is enough, which that we can also focus all the logic on the devtools package.
(I’ve tried reuse the vtui dev server,but it doesn’t work fine).

Reply:
yeah

If we move this server to the cli we can probably ensure it's always created before the app starts. It would also ensure only one express app is created even when restarting the app (force reload that isn't yet implemented)